thriftbp/server.go

```diff
@@ -80,6 +85,10 @@ func NewServer(cfg ServerConfig) (*thrift.TSimpleServer, error) {
 		transport = cfg.Socket
 	}
 
+	transport = BaseplateWrapperTServerTransport{
+		transport, cfg.ReportServerConnectionCount,
+	}
```

Comment:
I think `false` should mean that you don't incur any overhead at all, so maybe something more like

```suggestion
  if cfg.ReportServerConnectionCount {
	  transport = CountedTServerTransport{transport}
	}
```